.devcontainer.json

```diff
@@ -0,0 +1,83 @@
+//////////////////////////////////////////////////////////////
+//
+// This file provides configuration options so that a PreTeXt 
+//   project can be edited and built using GitHub's Codespaces.
+// It is recommended to keep this in your repository even if you
+//   do not use this feature, as it will allow other to explore 
+//   your project easily.   
+// This file will be automatically generated by PreTeXt with the 
+//   latest updates unless you remove the first comment line above.
+//
+///////////////////////////////////////////////////////////////
+{
+  "name": "PreTeXt-Codespaces",
+
+  // This Docker image includes some LaTeX support, but is still not to large.  Note that if you keep your codespace running, it will use up your GitHub free storage quota.  Additional options are listed below.
+  // "image": "oscarlevin/pretext:small",
+  // If you need to generate more complicated assets (such as sageplots) or use additional fonts when building to PDF, comment out the above line and uncomment the following line.
+  "image": "oscarlevin/pretext:full",
+  // If you only intend to build for web and don't have any latex-image generated assets, you can use a smaller image:
+  // "image": "oscarlevin/pretext:lite",
+  "postCreateCommand": "sudo apt update && sudo apt install -y texlive-full --no-install-recommends && sudo apt install -y fonts-font-awesome --no-install-recommends",
```

Comment:
@oscarlevin this probably shouldn't be required when using your `pretext:full` image, but it is in order to build the guide PDF.

Reply:
Fixes errors `! LaTeX Error: File `musicography.sty' not found.` and `! I can't find file `FontAwesome'.`